docs/toml.md

```diff
@@ -718,7 +722,7 @@ $ curl -s "http://localhost:8080/api" | jq .
 - `/metrics`: You can enable Traefik to export internal metrics to different monitoring systems (Only Prometheus is supported at the moment).
 
 ```bash
-$ traefik --web.metrics.prometheus --web.metrics.prometheus.buckets="100,300"
+$ traefik --web.metrics.prometheus --web.metrics.prometheus.buckets="0.1,0.3"
```

Comment:
I'd keep this in line with the defaults.

Histograms only work when all the servers you're aggregating across have the same buckets, so by having this consistent it reduces the scope for mishaps.